lib/livebook/hubs/dockerfile.ex

```diff
@@ -157,6 +157,46 @@ defmodule Livebook.Hubs.Dockerfile do
     |> Enum.join("\n")
   end
 
+  @doc """
+  Builds Dockerfile definition for Livebook Agent app deployment.
+  """
+  @spec build_agent_dockerfile(config(), Hubs.Provider.t()) :: String.t()
+  def build_agent_dockerfile(config, hub) do
+    base_image = Enum.find(Livebook.Config.docker_images(), &(&1.tag == config.docker_tag))
+
+    image = """
+    FROM ghcr.io/livebook-dev/livebook:#{base_image.tag}
+    """
+
+    image_envs = format_envs(base_image.env)
+
+    hub_config = """
+    ARG TEAMS_KEY="#{hub.teams_key}"
```

Comment:
Technically none of these need to be ARGs, they could just be runtime ENVs. The reason we have TEAMS_KEY as arg in the other file, is because of `warmup_apps.sh`, which boots Livebook at Docker build time and may require secrets.